pymbolic/interop/ast.py

```diff
@@ -211,27 +221,31 @@ def map_Call(self, expr):
         else:
             return p.Call(func, args)
 
-    def map_Num(self, expr):
-        # (object n) -- a number as a PyObject.
-        return expr.n
+    if sys.version_info < (3, 14):
+        # FIXME: ast.Num is removed in Python 3.14
+        def map_Num(self, expr: ast.Num) -> Expression:  # pyright: ignore[reportDeprecated]
+            # (object n) -- a number as a PyObject.
+            return expr.n  # pyright: ignore[reportDeprecated]
 
-    def map_Str(self, expr):
-        return expr.s
+        # FIXME: ast.Str is removed in Python 3.14
+        def map_Str(self, expr: ast.Str) -> Expression:  # pyright: ignore[reportDeprecated]
+            return expr.s  # pyright: ignore[reportDeprecated]
 
-    def map_Bytes(self, expr):
-        return expr.s
+        # FIXME: ast.Bytes is removed in Python 3.14
+        def map_Bytes(self, expr: ast.Bytes) -> Expression:  # pyright: ignore[reportDeprecated]
+            return expr.s  # pyright: ignore[reportDeprecated]
```

Comment:
All of these seem to be removed in Python 3.14. As far as I can tell, we can keep the `map_` methods anyway, but would need to define some dummy classes for typing. What's preferred?